### PR TITLE
Import unity dll dependency

### PR DESCRIPTION
**Pull request by Atralupus**
Import UnityEngine, UnityEngine.CoreModule dll

Current method is temporary...
1. Install Unity Version 2021.3.0f1
2. Copy `UnityEngine.dll` and `UnityEngine.CoreModule.dll` FYI https://docs.unity3d.com/Manual/UsingDLL.html
3. Paste in Libplanet.Unity/UnityReference/~

### CODE REVIEW

**Comment by Atralupus** [comment]
> Can we also add `UnityEditor.dll` and `UnityEditor.CoreModule.dll` references? 🙂

I was worried license issues
Is there any problem?😭

**Comment by greymistcube** [comment]
> I was worried license issues

Is there a difference between `UnityEngine.dll` and `UnityEditor.dll`? I thought it'd be okay as long as we don't actually include `dll` files. In any case, @longfin might know better about this. 😶

**Comment by Atralupus** [comment]
> 2\. UnityEngine

I was mistaken... because I saw `UnityEditor.dll` as `UnityEngine.dll`
So I thought you asked me to upload the `UnityEngine.dll`
All right, Ill add `UnityEditor.dll` and `UnityEditor.CoreModule.dll`👍

**Comment by greymistcube** [comment]
Also, as this will eventually be merged to `main`, the following check needs to be resolved. 😶

```
Run hooks/pre-commit
  hooks/pre-commit
  shell: /bin/bash -e {0}
  env:
    DOTNET_ROOT: /home/runner/.dotnet
The project Libplanet.Unity is not documented in the CONTRIBUTING.md file.
Error: Process completed with exit code 1.
```

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,6 +115,9 @@ on GitHub consists of several projects:
  -  *Libplanet.Explorer.Executable*: Turns Libplanet Explorer into a single
     executable binary so that it is easy to distribute.
 
+ -  *Libplanet.Unity*: Libplanet unity plugin
+    This is a temporary plugin and will be transferred to sdk later.
```
Comment (greymistcube) [inline CONTRIBUTING.md:119]:
```suggestion
 -  *Libplanet.Unity*: User-friendly pluggable API for developing and running
    a *Libplanet* blockchain node within Unity.
```
```

Comment (greymistcube) [reply CONTRIBUTING.md]: I don't think phrases such as "temporary" and "will be moved later" are appropriate. 😶

File: Libplanet.Unity/Libplanet.Unity.csproj [hunk 2]
```diff
@@ -83,4 +72,19 @@
...
+    <Reference Include="UnityEngine.CoreModule">
+        <HintPath>.\UnityReference\UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEditor">
+        <HintPath>.\UnityReference\UnityEditor.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEditor.CoreModule">
+        <HintPath>.\UnityReference\UnityEditor.CoreModule.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```
Comment (greymistcube) [inline Libplanet.Unity/Libplanet.Unity.csproj:89]:
```suggestion
  <ItemGroup>
    <Reference Include="UnityEngine">
      <HintPath>.\UnityReference\UnityEngine.dll</HintPath>
    </Reference>
    <Reference Include="UnityEngine.CoreModule">
      <HintPath>.\UnityReference\UnityEngine.CoreModule.dll</HintPath>
    </Reference>
    <Reference Include="UnityEditor">
      <HintPath>.\UnityReference\UnityEditor.dll</HintPath>
    </Reference>
    <Reference Include="UnityEditor.CoreModule">
      <HintPath>.\UnityReference\UnityEditor.CoreModule.dll</HintPath>
    </Reference>
  </ItemGroup>
```

File: Libplanet.Unity.Tests/Libplanet.Unity.csproj [hunk 3]
```diff
@@ -77,4 +77,13 @@
...
   </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="UnityEngine">
+      <HintPath>.\UnityReference\UnityEngine.dll</HintPath>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule">
+        <HintPath>.\UnityReference\UnityEngine.CoreModule.dll</HintPath>
+    </Reference>
+  </ItemGroup>
```
Comment (greymistcube) [inline Libplanet.Unity.Tests/Libplanet.Unity.csproj:88]:
```suggestion
  <ItemGroup>
    <Reference Include="UnityEngine">
      <HintPath>.\UnityReference\UnityEngine.dll</HintPath>
    </Reference>
    <Reference Include="UnityEngine.CoreModule">
      <HintPath>.\UnityReference\UnityEngine.CoreModule.dll</HintPath>
    </Reference>
  </ItemGroup>
```

Just being picky. 🙃

Comment (Atralupus) [reply Libplanet.Unity.Tests/Libplanet.Unity.csproj]: I pasted it spaces were added... 😭

**Comment by greymistcube** [comment]
One last thing, can you squash last few commits for applying minor suggestions? 😅

**Comment by Atralupus** [comment]
> One last thing, can you squash last few commits for applying minor suggestions? 😅

Oops, All right

**Review by greymistcube — APPROVED** [review]

**Pull request merged**